CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 
+## 5.2.3 - 2020-09-29
+### Changed
+- Increased DX (Developer Experience) and modified output of DumpRoleHierarchyCommand.
```

Comment:
Als je de noodzaak voelt om uit te leggen wat DX betekend, waarom haal je de afkorting dan niet weg?  Je herhaald dit toch niet later in de text...

Reply:
Nee maar in de context van dit document is het de eerste keer dat de term gebruikt wordt, een uitleg wat het betekent is dan gangbaar volgens mij. In toekomstige verbeteringen kan alleen `DX` worden gebruikt omdat de verklaring van wat het is al gegeven is in dit stukje.